api_server.py

```
@@ -75,98 +85,97 @@ def format_hours(minutes: Optional[int]) -> Optional[str]:
+    col = "crew_id" if table == "crew_availability" else "appliance_id"
+    conn = get_db()
+    curr = conn.execute(
+        f"SELECT end_time FROM {table} WHERE {col} = ? AND start_time <= ? AND end_time > ? LIMIT 1",
+        (entity_id, now, now),
+    ).fetchone()
+    return _format_avail_info(parse_dt(curr["end_time"]) if curr else None, now)
 
 
 def get_weekly_stats(crew_id: int) -> Dict:
     start, end = get_week_boundaries()
     now = datetime.now()
```

Ruff (0.15.7)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)